### ISSUE TITLE
MUON FRONT END: Updates for the Momentum Slits

### ISSUE THREAD

**Issue by KathrynBaker** [ISISComputingGroup/IBEX#1797]
During the testing/setup a number of changes were made on the fly to get them working which are detailed here:

Change 1 - Add a galil program for the jaws
Whilst only an interim measure, as the plan is to get the Muon instrument slits working under EPICS, the initial setup requires them to remain controlled via the galil program already used in SECI. As such there needs to be an addition of a file called `galil_Muon_Slits.gmc` in EPCIS\support\galil\master\gmc with the code to control the slits (currently in place on NDEMUONFE, or available within LabVIEW Modules).

Change 2 - Ensure the program is being used
It might also be wise to add a copy of the galil1.cmd on NDEMUONFE to the default examples currently in EPICS\support\barndoors\master\settings, which has a GalilStartController line which does what is needed at this point

Change 3 - Corrections to the db file
The momentum_slits.db (in EPICS\support\barndoors\master\db\) had a few changes made:
- the PVs were altered to be called MSLITS rather than MOMENTUMSLITS (just for PV name lengths to be kept shorter)
- The two aliases were written as `$(P):M...`, there shouldn't be a `:` in there it should just be `$(P)M...`
- It would be a good idea to rename the convert PVs as well to something shorter and in keeping with the naming conventions, such as MTOGAP and MFRMGAP - although this hasn't at present been done on NDEMUONFE

**Comment by KathrynBaker** [comment]
Pull request for change 1:
https://github.com/ISISComputingGroup/EPICS-galil/pull/6

Pull Request for changes 2 and 3:
https://github.com/ISISComputingGroup/EPICS-barndoors/pull/3

**Comment by AdrianPotter** [comment]
Please could you add some documentation for setting up the Jaws on MFE in the style of:

https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Barndoors-and-Momentum-Slits-on-MUON-Front-End

As it stands, I don't know how to properly set up the jaws with a Galil. It'd be better if you weren't the only person who knows how they work. Not helped by #1617

**Comment by KathrynBaker** [comment]
I’m not sure I understand what you’re asking for. The jaws I set up are the momentum slits on that page (so I should correct the naming), the rest of the instructions are actually available in the Release Notes for Release 2.1.0, and at the bottom of that page, so for setup it is just a copy and paste of the pre created files, which is referred to in the Setup section on the page you linked to. All I did, was copy those files, (add a galil1.cmd) and create a block (which I discovered need to point to the .RBV field in the record for an updating value), and is a more generic motor behaviour. For what I did, this page is the instructions.
